Core/src/Geometry/CylinderVolumeStack.cpp

```diff
@@ -923,13 +961,15 @@ void CylinderVolumeStack::update(
     ACTS_VERBOSE("Commit and update outer vector of volumes");
     m_volumes.clear();
     for (auto& vt : volumeTuples) {
-      vt.commit();
+      vt.commit(logger);
       m_volumes.push_back(vt.volume);
     }
   }
 
   m_transform = newVolume.globalTransform;
-  Volume::update(std::move(newBounds));
+  // @TODO: We probably can reuse m_transform
+  m_groupTransform = m_transform;
```

Comment:
This is the actualy bugfix. The TODO is left for follow up work (should only reduce memory footprint slightly.)